Decisions.Microsoft365.Exchange/InputExchangeSettings.cs

```diff
@@ -0,0 +1,55 @@
+using System.ComponentModel;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Properties;
+using DecisionsFramework.Design.Properties.Attributes;
+
+namespace Decisions.Microsoft365.Exchange;
+
+public class InputExchangeSettings: ICreateChildEntityInFolder
```

Comment:
If this becomes the new Exchange Settings Object, I think we need the ORM Entity attribute that we had in `ExchangeSettings.cs`, right?

Reply:
The intent behind this is more like the SmtpServerSettings object. The step will hold an instance of this object unique to the step it's configured on

Reply:
Discussed this with @noah-decisions, basically we are overriding the inputs using InputExchangeSettings, just like it's being done on SmtpServerSettings (not an ORM entity) that overrides SMTP Settings (which are actually this object as a property of the settings object).

ExchangeSettings is still the settings object for the module and InputExchangeSettings is just overriding the input to the steps. If the user doesn't define it, the steps should use the settings saved to the other(ExchangeSettings) object. Making our new class(InputExchangeSettings ) an ORM object would just cause more issues